openproblems/tasks/_batch_integration/batch_integration_graph/datasets/immune_hm.py

```diff
@@ -0,0 +1,37 @@
+from .....data.immune_cells_human_mouse import load_immune_hm
+from .....tools.decorators import dataset
+
+
+@dataset(
+    dataset_name="Immune (human/mouse)(by batch)",
+    data_url=load_immune_hm.metadata["data_url"],
+    data_reference=load_immune_hm.metadata["data_reference"],
+    dataset_summary="Human and Mouse immune cells from peripheral blood and bone marrow "
+    "taken from multiple datasets across technologies (10X, "
+    "Smart-seq2).",
+    image="openproblems",
+)
+def immune_hm_batch(test=False):
+    import scanpy as sc
+
+    adata = load_immune_hm(test)
+    adata.uns["organism"] = "human"
+    adata.obs["labels"] = adata.obs["final_annotation"]
```

Comment:
May need to add an `adata.obs['batch']` assignment here too. I'm not sure that batch is labeled as such in all of these datasets.

Reply:
batch seems to be named correctly already, please add a comment that it's already assigned in the loaded data.

Reply:
it's called batch